File: apps/problems/kth-largest-element-in-an-array/boilerplate-full/function.java

    import java.io.*;
    import java.util.*;

    

    //##USER_CODE_HERE##

    public class Main {
      
    public static void main(String[] args) throws IOException {
      BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
      int size_nums = Integer.parseInt(br.readLine().trim());
        int[] nums = new int[size_nums];
        String[] items_nums = br.readLine().trim().split("\\s+");
        for (int i = 0; i < size_nums; i++) {
          nums[i] = Integer.parseInt(items_nums[i]);
        }
int k = Integer.parseInt(br.readLine().trim());
      Solution obj = new Solution();
      int result = obj.findKthLargest(nums, k);
      System.out.println(result);
      br.close();
    }
      
    }
  

File: apps/problems/kth-largest-element-in-an-array/boilerplate-full/function.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>



//##USER_CODE_HERE


int main() {
    int m_nums;
std::cin >> m_nums;
std::vector<int> nums(m_nums);

for(int i = 0; i < m_nums; ++i) {
    std::cin >> nums[i];
}
int k;
std::cin >> k;
    Solution obj;
    int result = obj.findKthLargest(nums, k);
    std::cout << result << std::endl;
    return 0;
}
